tests_new/integration_tests/modules/dashboards/test_dashboard.py

```diff
@@ -0,0 +1,113 @@
+from assertpy import assert_that
+
+from integration_tests.modules.dashboards.queries import (
+    search_dashboards,
+    get_dashboard,
+    list_dashboard_shares,
+    get_author_session,
+    get_reader_session,
+)
+from integration_tests.modules.dashboards.mutations import (
+    update_dashboard,
+    delete_dashboard,
+    approve_dashboard_share,
+    reject_dashboard_share,
+)
+from integration_tests.modules.dashboards.conftest import create_dataall_dashboard
+from integration_tests.errors import GqlError
+
+UPDATED_DESC = 'new description'
+
+
+def test_get_author_session(client1, persistent_env1):
+    assert_that(get_author_session(client1, persistent_env1.environmentUri)).starts_with('https://')
+
+
+def test_get_author_session_unauthorized(client2, persistent_env1):
+    assert_that(get_author_session).raises(GqlError).when_called_with(client2, persistent_env1.environmentUri).contains(
+        'UnauthorizedOperation', 'CREATE_DASHBOARD'
+    )
+
```

Comment:
I have not tested yet, but the `DashboardQuicksightService.get_quicksight_author_session` is not decorated. Instead it uses `DashboardQuicksightService._check_user_must_be_admin` which uses `TenantPolicyRepository.is_tenant_admin` where is_tenant_admin does not exist. I am not sure how this is working at the moment

Reply:
`get_quicksight_author_session` is resolver for `getPlatformAuthorSession` (only used in Admin Monitoring view if enabled) while `get_quicksight_designer_url` is resolver for `getAuthorSession` (used for start QS Session for any user)